scripts/zones/Horlais_Peak/mobs/Chlevnik.lua

```diff
@@ -1,17 +1,92 @@
 -----------------------------------
 -- Area: Horlais Peak
---  Mob: Chlevnik
--- KSNM99
+-- Mob: Chlevnik
+-- KSNM99 : Horns of War
+
+-- TODO : Update Howl to give 25% Attack instead of 15% - Update Meteor to 1.6 fTP + dINT * 3(!)
 -----------------------------------
----@type TMobEntity
 local entity = {}
 
 entity.onMobSpawn = function(mob)
-    mob:addImmunity(xi.immunity.SILENCE)
+    mob:setAnimationSub(3)
+    mob:setUnkillable(true)
+    mob:setMod(xi.mod.TRIPLE_ATTACK, 5)
+    mob:addMod(xi.mod.STUNRES, 90)
+    mob:addMod(xi.mod.SLEEPRES, 90)
+    mob:setMod(xi.mod.DELAY, -1000)
```

Comment:
Seeing some immunities that can be added based off of captures into onMobInititalize:

```
mob:addImmunity(xi.immunity.DARK_SLEEP)
mob:addImmunity(xi.immunity.LIGHT_SLEEP)
mob:addImmunity(xi.immunity.PETRIFY)
mob:addImmunity(xi.immunity.SILENCE)
```

Along with deleting the SLEEPRES mod